homeassistant/components/playstation_network/helpers.py

```diff
@@ -38,7 +38,7 @@ class PlaystationNetworkData:
     presence: dict[str, Any] = field(default_factory=dict)
     username: str = ""
     account_id: str = ""
-    available: bool = False
+    available: str = "unavailable"
```

Comment:
should we also rename this to availability now, since it has a different meaning than "available"?